src/styles/UserInput.css

```diff
@@ -0,0 +1,53 @@
+* {
+  font-size: 20px;
+}
+
+.background-picture {
+  background-image: url("startPage.jpg");
```

Comment:
Picture not showing up when you load the page. Is it a broken path? =S

/Martin & Etna